ouroboros-consensus/test-consensus/Test/Dynamic/Network.hs

```diff
@@ -35,9 +39,10 @@ import           Control.Monad.Class.MonadThrow
 import           Control.Monad.Class.MonadTime
 import           Control.Monad.Class.MonadTimer
 
-import           Network.TypedProtocol.Codec (AnyMessage (..))
 import           Network.TypedProtocol.Channel
+import           Network.TypedProtocol.Codec (AnyMessage (..))
 
+import           Ouroboros.Network.AnchoredFragment as AF
```

Comment:
I think you forgot `qualified` (or was that on purpose)?

Reply:
Done. Sort of on purpose. But removed to avoid confusion.